backend/molgenis-emx2-io/src/test/java/org/molgenis/emx2/io/TestCohortCatalogueMultipleSchemas.java

```diff
@@ -59,80 +54,4 @@ public void importTest() throws IOException {
     //        cohortsSchema);
     assertEquals(48, TestCohortCatalogueMultipleSchemas.cohortsSchema.getTableNames().size());
   }
-
-  @Test
-  public void importTest2() throws IOException {
-    // load data model
-    SchemaMetadata schema =
-        Emx2.fromRowList(CsvTableReader.read(new File("../../data/datacatalogue/molgenis.csv")));
-    conceptionSchema.migrate(schema);
-
-    loadSchema("../../data/datacatalogue/Conception.xlsx", conceptionSchema);
-    assertEquals(48, TestCohortCatalogueMultipleSchemas.conceptionSchema.getTableNames().size());
-  }
-
-  private void loadSchema(String fileName, Schema schema) throws IOException {
-    File f = new File(fileName);
-    Path file = f.toPath();
-    if (!f.exists()) {
-      ClassLoader classLoader = getClass().getClassLoader();
-      file = new File(classLoader.getResource(fileName).getFile()).toPath();
-    }
-
-    TableStoreForXlsxFile store = new TableStoreForXlsxFile(file);
-    if (store.containsTable("molgenis")) {
-      SchemaMetadata source = Emx2.fromRowList(store.readTable("molgenis"));
-      source.setDatabase(schema.getDatabase()); // enable cross links to existing data
-      System.out.println(source);
-      StopWatch.print("schema loaded, now creating tables");
-
-      database.tx(
-          db -> {
-            schema.migrate(source);
-          });
-    }
-
-    SchemaMetadata source = schema.getMetadata();
-    // don't put alter in same transaction as update
-    database.tx(
-        db -> {
-          runImportProcedure(store, source, schema);
-          StopWatch.print("import of data complete");
-        });
-
-    // repeat for idempotency test (should not change anything)
-    database.tx(
-        db -> {
-          schema.migrate(source);
-        });
-
-    database.tx(
-        db -> {
-          runImportProcedure(store, source, schema);
-          StopWatch.print("import of data complete");
-        });
-
-    // verify export doesn't throw exceptions
-    Path tempDir =
```

Comment:
Do you cover these tests somewhere else? I can imagine the export test would be valuable to have?

Reply:
There are import/export roundtrip tests at https://github.com/molgenis/molgenis-emx2/blob/master/backend/molgenis-emx2-webapi/src/test/java/org.molgenis.emx2.web/TestWebApi.java

Reply:
we could add a few test assertions to verify that state is also correct